spec/features/post/post_show_spec.rb

```diff
@@ -0,0 +1,68 @@
+require 'rails_helper'
+
+RSpec.describe 'Post Show', type: :system do
+  before(:each) do
+    # Create users
+    @user_first = User.create(name: 'Anthony', photo: 'https://images.pexels.com/photos/1108099/pexels-photo-1108099.jpeg?auto=compress&cs=tinysrgb&w=800', bio:
+    'Lorem ipsum dolor sit amet sss labore fugiat!.', posts_counter: 1)
+    @user_second = User.create(name: 'Sunday', photo: 'https://images.pexels.com/photos/1108099/pexels-photo-1108099.jpeg?auto=compress&cs=tinysrgb&w=800', bio:
+    'Lorem ipsum dolor sit amet sss labore fugiat!.', posts_counter: 2)
+    @user_third = User.create(name: 'Ezema', photo: 'https://images.pexels.com/photos/1108099/pexels-photo-1108099.jpeg?auto=compress&cs=tinysrgb&w=800', bio:
+    'Lorem ipsum dolor sit amet sss labore fugiat!.', posts_counter: 3)
+    @post_one = Post.create(author_id: @user_first.id, title: 'The jungle', text: 'Lorem ipsum fugiat!', likes_counter: 1,
+                            comments_counter: 1)
+    @post_two = Post.create(author_id: @user_first.id, title: 'The animals', text: 'Lorem ipsum fugiat!',
```

Comment:
- [ ] It would be greatly appreciated if you could also make sure that these test cases pass before resubmitting the project for review. Your attention to detail and thorough testing will help ensure the success of this project. kindly make that these tests below passed the ask for review again :point_down: 

- Tests that assert you can see the username of all other users.
- Tests that asserts you can see a section for pagination if there are more posts than fit on the view.

Reply:
@Alfredbis29 Thank you for reviewing our project. I will work on it asap